components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/internal/AccountServiceComponent.java

```diff
@@ -63,6 +66,17 @@ protected void activate(ComponentContext context) {
         if (log.isDebugEnabled()) {
             log.debug("AccountLockService is registered");
         }
+        AccountLockTenantMgtListener accountLockTenantMgtListener = new AccountLockTenantMgtListener();
+        context.getBundleContext().registerService(TenantMgtListener.class, accountLockTenantMgtListener, null);
+        if (log.isDebugEnabled()) {
+            log.debug("AccountLockTenantMgtListener is registered");
+        }
+        try {
+            AccountServiceDataHolder.getInstance().getRealmService().getBootstrapRealm().
+                    getUserStoreManager().addRole(AccountConstants.ACCOUNT_LOCK_BYPASS_ROLE, null, null, false);
+        } catch (org.wso2.carbon.user.api.UserStoreException e) {
+            log.error(String.format("Error in saving role in bootstrap realm: %s", e.getMessage()));
```

Comment:
log.error(String.format("Error while saving role: %s in bootstrap realm.", AccountConstants.ACCOUNT_LOCK_BYPASS_ROLE), e);